colorpicker-build.json

```diff
@@ -28,7 +28,15 @@
         "executableName": "colorpicker-app"
     },
     "snap": {
-        "plugs": ["default", "desktop"]
+        "confinement": "classic",
+        "plugs": ["default", "libxkbcommon-x11-0"],
+        "buildPackages" : [
+            "python2.7", "make", "gcc", "libxtst-dev", "libpng++-dev", "libxkbcommon-x11-dev"
+        ],
+        "stagePackages": [
+            "default",
+            "libxkbcommon-x11-0"
+        ]
```

Comment:
would snapcraft use the colorpicker-build.json file?
I think it needs to be added to snapcraft.yaml

Reply:
The best would be to use only electron-builder. their api is available here for snapcraft : https://www.electron.build/configuration/snap
otherwise, why not do the opposite, but I don't know enough about it yet

Reply:
ок